boba-docs/.docusaurus/client-modules.js

```diff
@@ -0,0 +1,6 @@
+export default [
```

Comment:
are we sure these imports work in the pipeline or whereever these are needed? Since they seem to access your local device.

Reply:
+1 seems like these should be done as relative paths if possible?

Reply:
Everything in the `.docusaurus/` dir is temp files autogenerated based mainly on contents in `docusaurus.config.js`. I'm not sure which piece of config generates this specifically, but the js file is where we should look.